client/config/crd/objectstorage.k8s.io_bucketaccessclasses.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.16.4
+    controller-gen.kubebuilder.io/version: v0.17.3
```

Comment:
This change isn't ideal, but it seems to be because of https://github.com/kubernetes-sigs/container-object-storage-interface/pull/39

Dependabot updated the lib, but it doesn't know how to run `make generate`, so this didn't appear until my PR here. We might want to follow up with dependabot config to handle controller-gen differently in the future